WITH_ORACLE.md

```diff
@@ -81,14 +86,12 @@ If you want to use Oracle as source for your HOMS instance.
       password: <%= ENV['HOMS_DB_PASSWORD'] %>
     ```
 
-12. Run `docker-compose` with custom config:
+13. Run `docker-compose` with custom config:
 
     ```bash
     docker-compose -f docker-compose.dev.oracle.yml up -d
```

Comment:
I think we should use `docker-compose.dev.yml` here since we run HOMS locally. So `docker-compose.dev.yml` will be redundant.